packages/cx-flowcontainer/doc/FlowContainer.md

```diff
@@ -13,17 +13,7 @@ Details](/Components/FlowContainer/#content-editor-details).
 
 ## Site Builder Details
 
-### Available Tokens
-
-- `WithFullWidthConstraint`: Components with this Flow Container will always be full-width at any
-  viewport.
-- `WithTabletOneThirdConstraint`: Components will default to one-third-width upon addition on
-  tablet/desktop viewports, but can changed at either viewport if not desired. This token is useful
-  to apply if you expect that region to always use one-third-width components, to make content entry
-  faster and not resize each new component when added.
-- `ContentRegion` & `WithContentRegionVariations`: Provide tokens to nest Flow Containers.
-  - `ContentRegion`: A Flow Container designed to be nested inside another Flow Container.
-  - `WithContentRegionVariations`: A token to apply to nest the `ContentRegion`.
+### Usage
 
 ```js
```

Comment:
@misterjones  i would remove usage without context

Reply:
@hvanyo Ha, you beat me to it. 😆 

I was going to ask if this was still a useful example, and, if so, if you could help me with describing what it does.